templates/bot/ts/default/index.ts

```diff
@@ -38,8 +46,22 @@ const onTurnErrorHandler = async (context: TurnContext, error: Error) => {
 // Set the onTurnError for the singleton BotFrameworkAdapter.
 adapter.onTurnError = onTurnErrorHandler;
 
+// Define the state store for your bot.
```

Comment:
If L49 - L62 is not used in another place of index.js, can we encapsulate it into TeamsBot logic? The TeamsBot class handle if it is a command bot or dialog bot.